src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -1,45 +1,129 @@
 package seedu.address.model.util;
 
+import java.time.LocalDate;
 import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import seedu.address.model.CustomerManager;
+import seedu.address.model.Description;
+import seedu.address.model.DriverManager;
+import seedu.address.model.EventTime;
+import seedu.address.model.id.IdManager;
 import seedu.address.model.legacy.AddressBook;
 import seedu.address.model.legacy.ReadOnlyAddressBook;
 import seedu.address.model.person.Address;
+import seedu.address.model.person.Customer;
+import seedu.address.model.person.Driver;
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
 import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.TaskManager;
+import seedu.address.model.task.TaskStatus;
+import seedu.address.storage.CentralManager;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
```

Comment:
Thanks for taking the time to update this! Much needed PR!

Reply:
Oh Hmm.. The magic numbers though.. If the `IdManager` had the public method to `getId` and increment counter at the same time then there will not be any magic numbers.. But I cannot think of a better way to do this given the current impl.. Let's just leave it here

Reply:
abstracted to static variables